### ISSUE TITLE
Outdated scikit-learn API call in preprocessing.py

### ISSUE THREAD

**Issue by Shrecki** [mne-tools/mne-hcp#68]
Hello!

Using  function `hcp.preprocessing.apply_ref_correction(raw)` on HCP dataset returns the following error:
```python
File ~/miniconda3/envs/XXXXX/lib/python3.8/site-packages/hcp/preprocessing.py:100, in apply_ref_correction(raw, decim_fit)
     97 if len(ref_picks) == 0:
     98     raise ValueError('Could not find meg ref channels.')
--> 100 estimator = LinearRegression(normalize=True)  # ref MAG + GRAD
    101 Y_pred = estimator.fit(
    102     raw[ref_picks][0][:, ::decim_fit].T,
    103     raw[meg_picks][0][:, ::decim_fit].T).predict(
    104     raw[ref_picks][0].T)
    105 raw._data[meg_picks] -= Y_pred.T

TypeError: __init__() got an unexpected keyword argument 'normalize'
```

In preprocessing.py,  LinearRegression estimator is imported as:
```python
from sklearn.linear_model import LinearRegression

```

Looking through the API, the `normalize` parameter appears in version [0.18](https://github.com/scikit-learn/scikit-learn/blob/8c9c1f27b/sklearn/linear_model/_base.py#L502) but has been [removed since then](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html). Recommendation is to instead use a pipeline with a standard scaler, so line 93 should be replaced instead with for newer releases of scikit-learn to be compatible:

```python
from sklearn.linear_model import LinearRegression
from sklearn.preprocessing import StandardScaler
from sklearn.pipeline import Pipeline
estimator = Pipeline([('scaler', StandardScaler()), ('estimator', LinearRegression())])
```

**Comment by jasmainak** [comment]
@Shrecki can you make a PR?

**Comment by jasmainak** [comment]
closed by #69
